### PR TITLE
fixed layout typo in commands and added info re bootstrap

### PR DESCRIPTION
**Pull request by matthewhelmke**

### CODE REVIEW

File: src/en/commands.md [hunk 1]
```diff
@@ -1167,10 +1167,8 @@ Click on the expander to see details for each command.
 
              # How long to wait for a connection to the controller
              bootstrap-timeout: 600 # default: 10 minutes
-             # How long to wait between connection attempts to a controller 
-
```
Comment (matthewhelmke) [inline src/en/commands.md]: There were extra spaces in here that broke up the code block. I removed them. That is all.

**Comment by matthewhelmke** [comment]
@juju/docs
Ready for review.

The command page change was just a quick fix, nothing major.

I changed the order of the example commands on the bootstrap page, added other examples, and merged notes and examples together for each example.

**Comment by matthewhelmke** [comment]
Ready for review @juju/docs 
fixes #1047

**Comment by evilnick** [comment]
this is a good start for now. there are some areas we could still expand on, but I'll make another acrd for that at some point

**Pull request merged**